participants/yaellevy.json

```diff
@@ -2,7 +2,11 @@
     "name": "Yael Levi",
     "github": "yaellevy",
     "devto": "yaellevy",
+    "github_page": true,
     "posts": [
+        {
+            "url": "https://dev.to/yaellevy/session-3-create-github-pages-markdown-3hc3-temp-slug-964251/edit"
+        },
```

Comment:
Here is something the CI did not catch. This URL is incorrect. Please send a new pr fixing it.